src/index.js

```diff
@@ -2,27 +2,68 @@
 
 var MutationObserver = window.MutationObserver
 
+// @TODO, remove post testing
+var testGroups = [
+  {
+    name: 'old-timerz',
+    key: 'github.com/dino-dna/tagadactyl',
+    global: false,
+    users: [
+      { login: 'dylancwood' },
+      { login: 'rkelly' }
+    ]
+  },
+  {
+    name: 'def',
+    key: 'github.com/dino-dna/NOTTDACTYL',
+    global: false,
+    users: [
+      { login: 'swashcap' },
+      { login: 'wa11-e' }
+    ]
+
+  }
+]
 var tdac = {
-  _cache: {},
+  _cache: {
+    collaborators: {}, // GH collaborators, as stated by GH service. e.g. { 'dino-dna/tagadactyl': [...] }
+    groups: [] // user specified groups, e.g. [{ key: 'dino-dna/tagadactyl', name: 'mygroup', users: [...], global: boolean }]
+  },
+  get currentProjectKey () { return window.location.hostname + '/' + this.owner + '/' + this.repo },
+  get owner () { return window.location.pathname.split('/')[1] },
+  get repo () { return window.location.pathname.split('/')[2] },
+
+  bindClickConfigureGroupHandler (group) {
+    var node = document.getElementById(this.mapGroupToGroupId(group))
+    node.addEventListener('click', function (evt) {
+      evt.stopImmediatePropagation()
+      evt.stopPropagation()
+      evt.preventDefault()
+      console.log('blah!')
```

Comment:
Seeing this in the console:

```
Uncaught (in promise) TypeError: Cannot read property 'addEventListener' of null
    at Object.bindClickConfigureGroupHandler (chrome-extension://hjcmbbpfkobebchcbblbpcghbhneopog/index.js:38:9)
    at Object.<anonymous> (chrome-extension://hjcmbbpfkobebchcbblbpcghbhneopog/index.js:165:14)
    at Array.forEach (native)
    at Object.appendGroupInjectButton (chrome-extension://hjcmbbpfkobebchcbblbpcghbhneopog/index.js:163:16)
    at Array.forEach (native)
    at Object.injectGroups (chrome-extension://hjcmbbpfkobebchcbblbpcghbhneopog/index.js:162:30)
```

The error [originates here](https://github.com/dino-dna/tagadactyl/blob/207511e49fa8a6aa99304023bbe86a5e6b777207/src/index.js#L38). Maybe a race condition?